src/cdk-experimental/listbox/listbox.ts

```diff
@@ -105,6 +109,28 @@ export class CdkListbox<V> {
     items: this.items,
     textDirection: this.textDirection,
   });
+
+  /** Whether the listbox has received focus yet. */
+  private _touched = signal(false);
```

Comment:
I would probably not call this touched, that usually means that it has been focused _and then blurred_. Maybe just call it like `hasFocused`